Modules/ui.js

```diff
@@ -0,0 +1,41 @@
+import Store from './store.js';
+
+export default class UI {
+  static displayBooks() {
+    const books = Store.getBooks();
+    books.forEach((book) => UI.addBookToList(book));
+  }
```

Comment:
Kindly prioritize the use of `arrow` function instead of `class` specifically for this milestone 😄